### ISSUE TITLE
Parameter "activation" is ignored in _build_bert function.

### ISSUE THREAD

**Issue by anndop** [amaiya/ktrain#28]
I suggest, it should be like this:

`*** models.py.org       2019-11-14 13:11:35.732456424 +0000`
`--- models.py   2019-11-14 13:11:31.452468006 +0000`
`*************** def _build_bert(x_train, y_train, num_cl`
`*** 216,222 ****`
`                                      seq_len=maxlen)`
`      inputs = model.inputs[:2]`
`      dense = model.get_layer('NSP-Dense').output`
`!     outputs = Dense(units=num_classes, activation='softmax')(dense)`
`      model = Model(inputs, outputs)`
`      model.compile(loss=loss_func,`
`                    optimizer=U.DEFAULT_OPT,`
`--- 216,222 ----`
`                                      seq_len=maxlen)`
`      inputs = model.inputs[:2]`
`      dense = model.get_layer('NSP-Dense').output`
`!     outputs = Dense(units=num_classes, activation=activation)(dense)`
`      model = Model(inputs, outputs)`
`      model.compile(loss=loss_func,`
`                    optimizer=U.DEFAULT_OPT,`

**Comment by amaiya** [comment]
Thanks reporting this.  This is fixed in v0.6.1.
